modules/e2ee/E2EEContext.js

```diff
@@ -58,8 +59,9 @@ export default class E2EEcontext {
      *
      * @param {RTCRtpReceiver} receiver - The receiver which will get the decoding function injected.
      * @param {string} kind - The kind of track this receiver belongs to.
+     * @param {string} participantId - The (muc participant) id that this receiver belongs to.
```

Comment:
Let's note mention the fact that this is MUC related, since that might change at some point. For all intents and purposes this is just an identifier for the participant, and that's that.